apps/demos/Demos/FileUploader/CustomDropzone/jQuery/index.js

```diff
@@ -8,7 +8,16 @@ $(() => {
     uploadUrl: 'https://js.devexpress.com/Demos/NetCore/FileUploader/Upload',
     visible: false,
     onDropZoneEnter(e) {
-      if (e.dropZoneElement.id === 'dropzone-external') { toggleDropZoneActive(e.dropZoneElement, true); }
+      if (e.dropZoneElement.id === 'dropzone-external') {
+        const items = e.event.originalEvent.dataTransfer.items;
+
+        const isSingleFileDragged = items.length === 1;
+        const isValidFileExtension = `.${items[0].type.replace(/^image\//, '')}`;
```

Comment:
`is` prefix in the name says that this variable is a predicate which has true/false as a value, but use use a string as a value. Let's update the naming
```suggestion
        const draggedFileExtension = `.${items[0].type.replace(/^image\//, '')}`;
```